README.md

```diff
@@ -316,16 +316,17 @@ yarn test
 
 ### Releasing
 
-For ease the release process, we use `release-it` utility that helps with the process.
+For ease the release process, we use `release-it` utility that helps with the process and `towncrier` for creating and maintaining changelogs.
 
 https://www.npmjs.com/package/release-it
+https://towncrier.readthedocs.io/
 
 For using it and start a release you need to fulfill the requirements:
 
 - Have permissions to push on master branch
 - Have permissions on the @plone org on npmjs.com
```

Comment:
What kind of permission?

What do I need do to request such permission?

```suggestion
- Have permission on the [`@plone` organization on npm](https://www.npmjs.com/org/plone).
```

Reply:
What kind of permission? Write?

What do I need do to request such permission?